packages/at_persistence_secondary_server/lib/src/log/commitlog/at_commit_log.dart

```diff
@@ -127,12 +127,13 @@ class AtCommitLog extends BaseAtCommitLog {
   /// Returns the Iterator of [_commitLogCacheMap] from the commitId specified.
   @server
   Iterator<MapEntry<String, CommitEntry>> getEntries(int commitId,
-      {String? regex, int limit = 25}) {
+      {String? regex, int limit = 25, bool skipDeletes = false}) {
```

Comment:
I think we can make this a strategy.. where DefaultStrategy is to get all and SkipDeletesStrategy is to skip deletes, instead of if-else code might look more readable.